src/mcts/search.cc

```diff
@@ -600,7 +601,8 @@ void Search::EnsureBestMoveKnown() REQUIRES(nodes_mutex_)
                            : GetBestChildNoTemperature(root_node_, 0);
   final_bestmove_ = bestmove_edge.GetMove(played_history_.IsBlackToMove());
 
-  if (bestmove_edge.HasNode() && bestmove_edge.node()->HasChildren()) {
+  if (bestmove_edge.GetN() > 0 && bestmove_edge.HasNode() &&
```

Comment:
The `HasNode()` check is superfluous here (and should have been first).

Reply:
I'll remove it.